types/index.d.ts

```diff
@@ -70,6 +70,7 @@ declare namespace firebaseui.auth {
     signInSuccessUrl?: string;
     siteName?: string;
     tosUrl?: string;
+    privacyPolicyUrl?: string;
```

Comment:
Could you please also add it here after this line: https://github.com/firebase/firebaseui-web/blob/master/externs/firebaseui-externs.js#L257

Besides, we are having some issues with our Travis CI test. I've already submit a PR to disable it. Once that it's done, you can pull the latest changes from master branch and re-run the test. 

Thanks

Reply:
OK, I'll make a new pull request and close this one. Thanks for your kindness.